components/Article.jsx

```diff
@@ -10,6 +42,9 @@ const SingleArticle = ({ singleArticle }) => {
           <img src={singleArticle.article_img_url} />
           <p>{singleArticle.body}</p>
           <p>{singleArticle.author}</p>
+          <p>{articleVotes}</p>
+          <button onClick={handleVotes}>Like</button>
```

Comment:
The logic is in place to make these buttons not work once a vote has been registered, but it would be nice in this case for the button to also be disabled so our client has nice visual indication that they are disabled